cmd/hatchet-migrate/migrate/migrations/20251007231452_v1_0_48.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TYPE v1_payload_type ADD VALUE IF NOT EXISTS 'TASK_EVENT_DATA';
+ALTER TABLE v1_task_event
+    ADD COLUMN external_id UUID,
+    ADD COLUMN inserted_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP;
```

Comment:
We've double checked that this doesn't try to update all existing rows to `CURRENT_TIMESTAMP`? Seeing some contradictory information online: https://stackoverflow.com/questions/38916063/add-timestamp-column-with-default-now-for-new-rows-only